app/views/posts/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+<p>
+  <%= link_to 'Create new post', new_post_path, class: 'btn btn-primary' %>
+</p>
+<% @posts.each do |post|%>
+<div class="card container">
+  <div class="card-body ">
+    <h5 class="card-title"><%=post.title%></h5> <% if signed_in? %> <small class="text-muted">Author: <%= post.user.username %></small>
```

Comment:
**REQUIRED** 
* [ ] There should be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright